src/components/Hexagon/Cube.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { CSSProperties } from "react"
+/** @jsxImportSource preact */
+import { CSSProperties } from "react"
```

Comment:
React vs Preact?

Reply:
React type for the interface but preact used for the component. Removing the preact declaration gives us issues, and We don't get any build or typecheck issues with this, but let me know if you have ideas